### PR TITLE
List all students

### PR DESCRIPTION
**Pull request by rhnonose**
I created a separate controller/view/template for the students to not clump the user controller too much and to separate the permissions as well when it comes to placing it, but I'm not too sure about it.
Thoughts @ggpasqualino @ghatighorias ?

### CODE REVIEW

**Comment by ggpasqualino** [comment]
Having a separated controller/view/template is very reasonable to me 👍

File: web/controllers/student_controller.ex [hunk 1]
```diff
@@ -0,0 +1,12 @@
+defmodule CoursePlanner.StudentController do
+  use CoursePlanner.Web, :controller
+  alias CoursePlanner.User
+  alias CoursePlanner.Router.Helpers
+  import Ecto.Query
+
+  def index(conn, _params) do
+    query = from u in User, where: u.role == "Student"
+    students = Repo.all(query)
```
Comment (ggpasqualino) [inline web/controllers/student_controller.ex]: you are missing the `deleted_at` condition here

**Comment by ghatighorias** [comment]
As I can see you don't show the status of the students. but in general . it's not a bad idea to have it there

File: web/controllers/student_controller.ex (hunk 1 above)
Comment (rhnonose) [inline web/controllers/student_controller.ex]: Yeah right.

**Comment by rhnonose** [comment]
I'll include the student status when putting the status change actions.

**Pull request merged**